docs/pages/deployment/audit-logging.rst

```diff
@@ -0,0 +1,19 @@
+.. _audit-logging:
+
+Audit Logging
+#############
+
+.. note ::
+    This feature is under development, not all relevant operations might be included in the audit log.
+
+Important events are logged as audit events. Examples are creation of a new cryptographic key pair or its usage when signing or decrypting.
+
+Audit events are logged to application log and can be recognized by the ``log`` field being set to ``audit``.
+In addition, the events contain the following fields:
+
+- ``operation`` which contains name action that was performed
+- ``actor`` which contains the name of the user/system that performed the action.
+  Actors from inside the application (e.g. triggered by a scheduled event) are prefixed with ``app-``.
```

Comment:
I don't know how valuable these are?

Reply:
I'd say we need to log _some_ actor indicating it's from an internal app process. But maybe that should be indicated by an `interface` field? (e.g. `HTTP` or `App`)?